flink-kubernetes-webhook/pom.xml

```diff
@@ -96,6 +96,12 @@ under the License.
             <version>${okhttp.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
```

Comment:
We're trying to avoid using mockito in the project. We write test classes instead.

Reply:
@morhidi Understand, I looked at using https://junit-pioneer.org/docs/environment-variables/ which has some nice capabilities to change system env vars, but noticed when running it I get `This extension uses reflection to mutate JDK-internal state, which is fragile` warnings when running in intellij. Do you think it would be ok to add a method to EnvUtils e.g. setEnvironment(Map env) and use this to override System env vars ? Or can you think of something better ?

Reply:
Actually noticed there is a method in TestUtils, am I ok to use that ?

Reply:
@gyfora what do you think? Shall we lift the 'mockito' restriction? It's quite convenient to use the EnvUtil how it is implemented today.

Reply:
Ah ok, I didn't realize there's `TestUtils.setEnv(Map<String, String> newEnv)`, yes you can use that.